todo/migrations/0011_auto_20190423_2115.py

```diff
@@ -0,0 +1,90 @@
+# Generated by Django 2.2 on 2019-04-23 21:15
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('todo', '0010_attachment'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='task',
+            options={},
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='assigned_to',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='completed',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='completed_date',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='created_by',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='created_date',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='due_date',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='id',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='note',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='priority',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='task_list',
+        ),
+        migrations.RemoveField(
+            model_name='task',
+            name='title',
+        ),
+        migrations.CreateModel(
```

Comment:
need to move this step first, run data migration, and remove/recreate the Task table